### PR TITLE
OCPQE-12159:Invoke e2e test on HyperShift-Agent

### PR DESCRIPTION
**Pull request by LiangquanLi930**
[Hypershift hostedcluster] Invoke e2e test on HyperShift-Agent

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 1]
```diff
@@ -2222,6 +2222,36 @@ tests:
...
+    env:
+      E2E_RUN_TAGS: '@amd64 and @network-ovnkubernetes and @hypershift-hosted and
+        not @fips'
+      TAG_VERSION: '@4.13'
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~NonPreRelease&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&;~Serial&;~Disruptive&
+      TEST_TIMEOUT: "30"
+    test:
+    - chain: openshift-e2e-test-hypershift-qe
+    workflow: cucushift-installer-rehearse-baremetalds-assisted-hypershift-agent-guest
+- as: baremetalds-assisted-hypershift-agent-guest-p2-f14-destr
```
Comment (heliubj18) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:2240]: is it caused by the limitation of job name length?

**Comment by heliubj18** [comment]
LGTM

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml (hunk 1 above)
Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:2240]: yes

**Comment by LiangquanLi930** [comment]
@liangxia Could you help to review this PR? Thanks

**Pull request merged**